docs/docs/forms.mdx

```diff
@@ -446,13 +346,19 @@ to extract customs slots:
 1. Define a method `extract_<slot_name>` for every slot that should be mapped in a
    custom way.
 2. Make sure that in the domain file you list for your form only those slots that use 
```

Comment:
```suggestion
2. In your domain file, for your form's `requested_slots`, list only those slots that use
```

Is this still true? Can it not list slots with custom mappings under `required_slots` now?

Reply:
Both ways should work actually but decided to keep the old docs in since it's not necessary to migrate 2.x `FormValidationAction` to the new way - I've made a note to test this today actually to make sure both cases work and let you know.

Reply:
Yes, you can list slots with custom mappings under form `required_slots`, meaning you don't have to override the `required_slots` anymore, and it actually gives the extra benefit that you can use `utter_ask_<custom_slot>` for these slots now. However, you can continue implementing the 2.x way currently described in the docs too. How should we write that both ways work?

Reply:
Great that both ways do work! 
Since this is not the migration guide, my instinct is it should rather push users to the most sensible usage in 3.0, which is listing all required slots, custom and otherwise, in the domain, and only indicate the possibility of overwriting the method in the custom action when actual dynamic behaviour is required i.e. in the `dynamic form behaviour` below.